unit_tests/sources/declarative/test_manifest_declarative_source_dynamic_schema.py

```diff
@@ -0,0 +1,171 @@
+#
+#
```

Comment:
This does not have the correct file header. It should have a copyright notice and basic file-level docstring.